web/client/reducers/maps.js

```diff
@@ -243,23 +244,10 @@ function maps(state = {
         let newState = assign({}, state, {
             results: newMaps.map(function(map) {
                 if (map.id === action.mapId) {
-
```

Comment:
Hi, please can you see if you can add unit tests to cover these changes?

Reply:
just a refactor, no logic was changed